### PR TITLE
Fix merge of cluster template and cluster labels

### PR DESCRIPTION
**Pull request by brtkwr**
Additionally,
- apply `terraform fmt`
- improve magnum-tour/README.md

### CODE REVIEW

File: terraform/modules/cluster/main.tf [hunk 1]
```diff
@@ -12,23 +12,31 @@ resource "openstack_compute_keypair_v2" "keypair" {
...
+}
+
+resource "null_resource" "kubeconfig" {
+  triggers = {
+    kubeconfig = var.cluster_name
+  }
 
   provisioner "local-exec" {
-    command = "mkdir -p ~/.kube/${var.cluster_name}; openstack coe cluster config ${var.cluster_name} --dir ~/.kube/${var.cluster_name} --force"
+    command = "mkdir -p ~/.kube/${var.cluster_name}; openstack coe cluster config ${var.cluster_name} --dir ~/.kube/${var.cluster_name} --force; ln -sf ~/.kube/${var.cluster_name}/config ~/.kube/config"
```
Comment (JohnGarbutt) [inline terraform/modules/cluster/main.tf]: are you sure we should add that link to override the default in all directories?

File: terraform/modules/cluster/main.tf [hunk 2]
```diff
@@ -12,23 +12,31 @@ resource "openstack_compute_keypair_v2" "keypair" {
...
+  keypair       = openstack_compute_keypair_v2.keypair.id
+  flavor        = var.flavor_name
+  master_flavor = var.master_flavor_name
 
-  #TODO: fix how we can specify labels?
-  #labels = {
-  #  min_node_count                      = var.node_count
-  #  max_node_count                      = var.max_node_count
-  #}
+  labels = merge(data.openstack_containerinfra_clustertemplate_v1.clustertemplate.labels,
```
Comment (JohnGarbutt) [inline terraform/modules/cluster/main.tf:25]: very nice.

File: terraform/modules/cluster/main.tf (hunk 1 above)
Comment (brtkwr) [reply terraform/modules/cluster/main.tf]: AH i find it useful but I guess other users might find it annoying, I will get rid of it.

Comment (brtkwr) [reply terraform/modules/cluster/main.tf]: Done.

**Review by JohnGarbutt — APPROVED** [review]

**Pull request merged**